src/app/content/highlights/types.ts

```diff
@@ -46,7 +47,7 @@ export interface State {
   };
 }
 
-export type HighlightLocationFilters = Map<string, LinkedArchiveTree | LinkedArchiveTreeSection>;
+export type HighlightLocationFilters = LocationFilters;
```

Comment:
i don't think we should need a `HighlightLocationFilters` defined, we can use use `LocationFilters`

Reply:
I didn't want to produce the additional diff because of changing these names, but idc, it wasn't that much of the work